windows/src/engine/keyman32/tests/keyman32-tests/keyman32-tests.vcxproj

```diff
@@ -71,9 +72,6 @@
     <ProjectReference Include="..\..\keyman-engine.vcxproj">
       <Project>{a7069186-5ae7-4504-8926-861b7a34ba9e}</Project>
     </ProjectReference>
-    <ProjectReference Include="..\..\Keyman32.vcxproj">
-      <Project>{bd5564fb-35a5-4a3c-b96a-4a6578e2b593}</Project>
-    </ProjectReference>
```

Comment:
Interesting that the keyman32-tests no longer depend on Keyman32.

Reply:
For the `keyman32-tests` project to run the google tests it needed a static library for `keyman32`  so there is a new project called `keyman-engine` this is just a static version of `keyman32`. The  `keyman32-tests` includes this and no longer needs to reference `keyman32`.

Reply:
In the longer-term future, keyman32 may also depend on keyman-engine as a static library for the build.

Reply:
So probably keyman32-tests should be called keyman-engine-tests.

Reply:
Have made this rename